.github/workflows/ci-test.yml

```diff
@@ -27,7 +27,7 @@ jobs:
         with:
           fetch-depth: 0
       - name: Setup CI
-        uses: input-output-hk/catalyst-ci/actions/setup@master
+        uses: input-output-hk/catalyst-ci/actions/setup@tests/typhon-registration
```

Comment:
You should not change this because it refers to the master branch of this repo input-output-hk/catalyst-ci/actions/setup